### PR TITLE
Adding Jun's blog post

### PR DESCRIPTION
**Pull request by bennyvasquez**
Adding a blog post for Jun's intro video, and adding an open graph title for Jack's blog post, which I missed when I published that one.

### CODE REVIEW

**Comment by bennyvasquez** [comment]
https://d9c9a8e8.almalinux-org.pages.dev/blog/2024-09-23-meet-the-board-jun/

File: content/blog/2024-09-12-meet-the-board-jack.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ date: '2024-09-12'
 images:
   - /blog-images/2024/jack_qanda.png
 post:
-    title: ""
+    title: "Meet the board - Jack Aboutboul"
```
Comment (codyro) [inline content/blog/2024-09-12-meet-the-board-jack.md:12]: Wrong name

**Review by codyro — CHANGES_REQUESTED** [review]

**Review by codyro — APPROVED** [review]
I don't think that name/title is exposed anywhere, so it's not a blocker.

File: content/blog/2024-09-12-meet-the-board-jack.md (hunk 1 above)
Comment (bennyvasquez) [reply content/blog/2024-09-12-meet-the-board-jack.md:12]: that's on the Jack blog post, so I don't think it's wrong. Am I missing something?

**Pull request merged**